utility/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
+kt_android_library(
+    name = "utility_lib",
+    custom_package = "org.oppia.util",
+    srcs = glob(["src/main/java/org/oppia/util/**/*.kt"],
```

Comment:
When arrays have more than one entry each entry should be on its own line with correct continuation indentation.